config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -221,6 +221,39 @@ postsubmits:
     rerun_auth_config:
       github_users:
       - alvaroaleman
+  - name: post-test-infra-push-prow-staging
+    cluster: test-infra-trusted
+    # Runs on more than just the Prow dir to include some additional images that we publish to gcr.io/k8s-prow.
+    run_if_changed: '^(WORKSPACE|load.bzl|repos.bzl|containers.bzl|prow|ghproxy|label_sync/.+\.go|robots/commenter|robots/pr-creator|robots/issue-creator|testgrid/cmd|gcsweb)'
+    decorate: true
+    branches:
+    - ^master$
+    max_concurrency: 1
+    spec:
+      serviceAccountName: pusher
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220124-681f3531ec-test-infra
+        command:
+        - runner.sh
```

Comment:
What is the motivation for using the kubekins-e2e image and `runner.sh`?

Reply:
still need dind for building typescript

Reply:
now that you reminded me, updated the job to use dind